database/flush_db.py

```diff
@@ -0,0 +1,19 @@
+import redis
+import json
+import os
+
+config_path = 'config/default.json'
+
+with open(config_path, 'r') as config_file:
```

Comment:
It would be better to pass environment variables down through the Dockerfile or DockerCompose on the cron job.  You could even set up a script to run on the container start that first populates the environment from the config file.  Its just not great to have to open and read files a bunch.